io.github.julian_hochhaus.LG4X_V2.yaml

```diff
@@ -82,7 +82,7 @@ modules:
       - type: git
         url: https://github.com/Julian-Hochhaus/LG4X-V2
         commit: 70f37b1ce388c4fcd65a02ae6302f1ae032ee760
-        tag: v2.1.0
+        tag: 2.1.5
```

Comment:
you can't change a tag without changing the commit.

Reply:
Yep, I already learned, it's my first time working with flathub, do you have a comprehensive starter guide on how to publish a new release of a piece of software on flathub?

Actually, all problems were already fixed in another PR by @kevinsmia1939